src/welldev/io/DAO/Driver.java

```diff
@@ -0,0 +1,182 @@
+package welldev.io.DAO;
+
+
+import java.sql.*;
+
+public class Driver {
```

Comment:
Structure is not appropriate. You could create a class under util package and configure connection there. I can't see any DAO related code here. Please reconfigure.